samples/main.cpp

```diff
@@ -69,8 +69,11 @@ int main(int /*argc*/, char* /*argv*/[])
   auto console = NovelRT::LoggingService(NovelRT::Utilities::Misc::CONSOLE_LOG_APP);
   auto audio = runner.getAudioService();
   audio.lock()->initializeAudio();
-  //auto bgm = audio->load((soundsDirPath / "running.ogg").string(), true);
-  //auto jojo = audio->load((soundsDirPath / "jojo.ogg").string(), false);
+  auto bgm = audio.lock()->loadMusic((soundsDirPath / "marisa.ogg").string());
+  auto jojo = audio.lock()->loadSound((soundsDirPath / "caution.wav").string());
```

Comment:
Also, as a note: this is synchronous and causes delays for large files. Something well want to consider how to handle eventually